### ISSUE TITLE
Problem about prediction

### ISSUE THREAD

**Issue by Mffff666** [eriklindernoren/PyTorch-YOLOv3#636]
Does it possible to add another prediction item to the network?
For example, when training, I give the true value of an object's color such as 1 for red and let the network learn to predict the color of one object.
If it can work, how should I recode?

**Comment by Flova** [comment]
If I understand you correct you plan to predict not one class but two independent classifications?
You could implement this yourself but this is not trivial and you should understand the loss function of YOLO and you need to change parts in the whole pipeline (load two class labels, adjust loss function, ..). 
I would suggest that you could just encode the two classifications into one combined one.

The standard YOLO predicts one classification e.g.: `Dog`, `Cat`
If you also want to predict the color meaning e.g.: `Gray` and `Red`
You could just embed the second one it into the first one: `Gray_Dog`, `Red_Dog`, `Gray_Cat`, `Red_Cat`.

It is not the most sophisticated way, but it should work and its MUCH less effort.
